Framework/PythonInterface/mantid/utils/absorptioncorrutils.py

```diff
@@ -81,6 +88,7 @@ def __get_cache_name(meta_wksp_name, abs_method, cache_dirs=[], prefix_name=""):
                 "height": ws.run()['BL11A:CS:ITEMS:HeightInContainer'].lastValue(),
                 "sample_container": ws.run()['SampleContainer'].lastValue().replace(" ", ""),
                 "abs_method": abs_method,
+                "ms_method": ms_method,
```

Comment:
Add a check above this point that the `abs_method` is supplied when `ms_method` is specified.

Reply:
`abs_method` is a positional argument (i.e. mandatory by default) and `ms_method` is a keyword argument (optional with default to empty string).
There is no need to do any check here